src/main/java/com/ddang/usedauction/member/domain/Member.java

```diff
@@ -22,36 +23,39 @@
 @SQLRestriction("deleted_at IS NULL")
 public class Member extends BaseTimeEntity {
 
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
 
-    @Column(nullable = false)
-    private String memberId;
+  @Column(nullable = false)
+  private String memberId;
 
-    @Column(nullable = false)
-    private String passWord;
+  @Column(nullable = false)
+  private String passWord;
 
-    @Column(nullable = false)
-    private String email;
+  @Column(nullable = false)
+  private String email;
 
-    @Column
-    private boolean siteAlarm;
+  @Column
```

Comment:
```suggestion
  @Column
  @Enumerated(EnumType.STRING)
```
해당 어노테이션 추가 안할 시 enum 인덱스로 저장되는 것으로 압니다!

Reply:
넵 추가하겠습니다 !